project-communications/newsletter/newsletter-august21.md

```diff
@@ -0,0 +1,148 @@
+# Newsletter - August 2021
+
+**Hi everyone!!**
+
+Firstly, thank you for opening and reading this month's newsletter 😄 We are so happy you are here!
+
+We hope that everyone has enjoyed their summer and have been able to take time off if possible. 
+
+We are still seeing a lot of change in our world at the moment. 
+So, if anyone is struggling with these challenging times, we hope that the citizen science community can offer your support and connection.
+
+Email [Lotty](mailto:ccoupat@turing.ac.uk) to let us know if we can help you with involvement, and do feel free to share this newsletter to anyone who you think would be interested in joining us.
+
+They can sign up by following this link: https://tinyletter.com/AutisticaTuringCitizenScience.
+
+Best wishes for everyone! 💮
+
+# Project News
+
+The most recent updates on the project and it's progress can be found [here](https://github.com/alan-turing-institute/AutisticaCitizenScience/tree/master/project-management/project-updates).
+
+A lot of our resource has been going towards roadmapping the project and ensuring all our issues are up to date! 
+Our latest roadmap can be found in our github repository [here](https://github.com/alan-turing-institute/AutisticaCitizenScience/tree/master/project-management/project-roadmap).
+
+## Autistica Festival 
+
+Autistica held their annual festival last month 12-16 July. 
+
+They successfully uploaded all videos of the conference on their youtube page [here](https://www.youtube.com/playlist?list=PLg_hTtp1fDqo6CRmisAd21OKLnjG1xdQc).
```

Comment:
I would maybe change this to; "the whole conference is now availble to watch on their YouTube page"